### DISCUSSION TITLE
Why it shows I have 0 commits?

### DISCUSSION THREAD

**Discussion by MitkoVtori** [anuraghazra/github-readme-stats#2279] [Q&A] [answered]
Yesterday I added **include_all_commits=true** to my github stats card and everything was alright. 
Even before like 2 hours it was okay. Why now, it shows 0 commits, when I remove the **include_all_commits=true** tag it shows me my 2022 commits and when I add the tag they become 0 again?

**Comment by rickstaa** [comment, marked as answer]
@MitkoVtori, thanks for your question. This is likely caused by our PATs overloading and the fallback value of `0` being cached (see https://github.com/anuraghazra/github-readme-stats/issues/1471). This caching issue is solved when #2177 is merged. I would advise you to deploy your own Vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).
